styles/styles.css

```diff
@@ -335,4 +335,16 @@ main .section {
   .column-section > div.column-section-block > div:first-child > *:first-child {
     margin-top: 0;
   }
+
+  .column-section > div.column-section-block > div:first-child > div:first-child > *:first-child {
```

Comment:
I hope this would not affect L2 and L3 pages :)

Reply:
it will, this was an oversight(bug) in the column section style that became apparent while working on the sample page. Shall I send a separate PR for this line?

Reply:
Yes this can be a chore PR.

Reply:
Done, thanks.